docs/subset15.md

```diff
@@ -65,6 +65,7 @@ For programs that are in the subset, the compiler should conform to the Ballerin
    * error constructor `error(msg)`
    * `check` expression
    * `checkpanic` expression
+   * explicit anonymous function `function (T1 A1, T2 A2, ..., Tn An) { S1, S2, ..., Sn }`
```

Comment:
anonymous function with explicit parameter types
S1, ..., SN is unnecessary - just use ...

Reply:
Fixed